api/db/changelog/db.changelog-170-add-access-tier-id-into-institution-tables.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+  <changeSet author="yonghao" id="changelog-170-add-access-tier-id-into-institution-tables">
```

Comment:
Could you initialize these to registered tier and then set to non-null?

Reply:
changelog 150 has an example like this

Reply:
I plan to do the backfill in my dual write PR. So i just need to do this once.